Sources/swiftpm-xctest-helper/main.swift

```diff
@@ -0,0 +1,120 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+#if os(OSX)
+
+import XCTest
+import Foundation
+import Darwin.C
+
+enum Error: ErrorProtocol {
+    case invalidUsage
+    case unableToLoadBundle(String)
+    case couldNotOpenOutputFile(String)
+    case couldNotConvertStringToNSData(String)
+}
+
+func withRedirectedStdout<T>(body: () throws -> T) throws -> T {
+    fflush(stdout)
+    let oldFd = dup(1)
+    let newFd = open("/dev/null", O_WRONLY);
+    dup2(newFd, 1)
+    close(newFd)
+
+    let result = try body()
+
+    fflush(stdout)
+    dup2(oldFd, 1)
+    close(oldFd)
+
+    return result
+}
+
+extension String {
+    func normalizedPath() -> String {
+        var path = self
+        if !(path as NSString).isAbsolutePath {
+            path = NSFileManager.default().currentDirectoryPath + "/" + path
+        }
+        return (path as NSString).standardizingPath
+    }
+}
+
+extension NSFileHandle {
+    func writeUTF8String(_ string: String) throws {
+        guard let data = (string as NSString).data(using: NSUTF8StringEncoding) else {
+            throw Error.couldNotConvertStringToNSData(string)
+        }
+        write(data)
+    }
+}
+
+do {
+    guard Process.arguments.count == 3 else {
+        throw Error.invalidUsage
+    }
+    let bundlePath = Process.arguments[1].normalizedPath()
+    let outputFile = Process.arguments[2].normalizedPath()
+
+    // Bundle may write to stdout when it is being loaded, redirect that output to /dev/null.
+    let suite: XCTestSuite = try withRedirectedStdout {
+        guard let bundle = NSBundle(path: bundlePath) where bundle.load() else {
+            throw Error.unableToLoadBundle(bundlePath)
+        }
+        return XCTestSuite.default()
+    }
```

Comment:
Given that we now write to a file, I'm not sure there is a need to hide the output from loading the bundle? This seems like it might be interesting for diagnostics purposes, and the client can always choose to capture and hide the output?